core/src/main/java/io/undertow/server/handlers/builder/PredicatedHandlersParser.java

```diff
@@ -363,6 +363,11 @@ static Node parse(final String string, Deque<Token> tokens, boolean topLevel) {
                 if(token.getToken().equals(";") && tokens.peek()!=null && tokens.peek().getToken().equals(ELSE)) {
                     // something() -> predicate; ELSE predicate; - dont end processing since its followed by ELSE and its singular block
                     continue;
+                } else if (token.getToken().equals("\n") && tokens.peek() != null && isOperator(tokens.peek().getToken())) {
```

Comment:
This doesn't seem correct. What if there are multiple line breaks?

Reply:
not allowed. Only allowed at end of handlers section. Even in OG. 
First, tokenization turns /r into /n, secondly,  if you look at if above, it does not allow multiple breaks before ELSE( which is operator).

Reply:
OK